cmd/av/branch.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/spf13/cobra"
+)
+
+var branchCmd = &cobra.Command{
+	Use:     "branch",
+	Aliases: []string{"br"},
+	Short:   "managed branches",
+	Long: `managed branches
+
+Examples:
+  Create a new branch from base branch
+	   $ av branch <new branch name> <base branch name>
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 2 {
+			return fmt.Errorf("need to set new branch name and base branch name")
+		}
+
+		newBranch := args[0]
+		baseBranch := args[1]
+
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+		defaultBranch, err := repo.DefaultBranch()
+		if err != nil {
+			return errors.WrapIf(err, "failed to determine repository default branch")
+		}
+
+		db, err := getDB(repo)
+		if err != nil {
+			return err
+		}
+
+		tx := db.WriteTx()
+		defer tx.Abort()
+
+		if _, err := repo.CheckoutBranch(&git.CheckoutBranch{Name: baseBranch, NewBranch: false}); err != nil {
+			return err
+		}
+
+		if _, err := repo.CheckoutBranch(&git.CheckoutBranch{Name: newBranch, NewBranch: true}); err != nil {
```

Comment:
Do we need to back to the working branch? 
With the current implementation, it will checkout a new branch